pkgs/timezone/lib/src/tzdata/zone_tab.dart

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) 2014, the timezone project authors. Please see the AUTHORS
```

Comment:
Is this copyright OK?

Reply:
I believe so per our internal copyright list. New files would get the Dart license header, which is enforced via the license checker of the health workflow.